.github/workflows/main.yml

```diff
@@ -374,8 +374,7 @@ jobs:
         shell: bash
         run: |
           URL="https://api.github.com/repos/${{ github.repository }}/commits/${{ github.sha }}"
-          FILES=$(curl -s -X GET -G $URL | jq -r '.files[].filename')
-          if echo $FILES | grep -q "cpanfile"; then
+          if $(curl -s -X GET -G $URL | jq -r '.files[].filename' | grep -q "cpanfile"); then
```

Comment:
is this real shell code? If so, shouldn't that be something like:

```bash
if [ -n $(...) ] ; then
```

?

Reply:
Indeed